tools/syntax.txt

```diff
@@ -542,6 +542,38 @@ token keyword
 separated_list<IdentifierNameSyntax> identifiers
 token semi
 
+ClockingSkewSyntax kind=ClockingSkew
+token edge
+token hash
+ExpressionSyntax value
+
+ClockingDirectionSyntax kind=ClockingDirection
+token input
+ClockingSkewSyntax inputSkew
+token output
+ClockingSkewSyntax ouputSkew
+token inout
+
+ClockingItemSyntax kind=ClockingItem
+token defaultKeyword
+ClockingDirectionSyntax direction
+separated_list<AttributeSpecSyntax> assignments
+token semi
+MemberSyntax declaration
+
+ClockingDeclarationSyntax base=MemberSyntax kind=ClockingDeclaration
+token globalOrDefault
+token clocking
+token blockName
+token at
+ParenthesizedEventExpressionSyntax event
+token eventIdentifier
+token semi
+list<ClockingItemSyntax> items
+token endClocking
+token colon
```

Comment:
Should use NamedBlockClauseSyntax so that analysis code for that can be unified.